packages/p2p-v2/src/hooks/__tests__/useQueryString.spec.tsx

```diff
@@ -0,0 +1,226 @@
+const MockBuySell = () => <h1>Buy sell page</h1>;
+const MockMyProfile = () => {
+    const { deleteQueryString, queryString, replaceQueryString, setQueryString } = useQueryString();
+
+    return (
+        <div>
+            <button
+                data-testid='switch_payment_methods_btn'
+                onClick={() =>
+                    setQueryString({
+                        tab: 'Payment methods',
+                    })
+                }
+            >
+                Switch payment methods
+            </button>
+            <button
+                data-testid='switch_stats_btn'
+                onClick={() =>
+                    setQueryString({
+                        tab: 'Stats',
+                    })
+                }
+            >
+                Switch stats
+            </button>
+            <button
+                data-testid='pm_form_btn'
+                onClick={() =>
+                    setQueryString({
+                        form: 'Payment method form',
+                        tab: 'Payment methods',
+                    })
+                }
+            >
+                Switch payment method form
+            </button>
+            <button
+                data-testid='counterparties_btn'
+                onClick={() =>
+                    replaceQueryString({
+                        form: 'Payment method form',
+                        tab: 'Counterparties',
+                    })
+                }
+            >
+                Switch counterparties
+            </button>
+            <button data-testid='go_back_btn' onClick={() => deleteQueryString('tab')}>
+                Go back
+            </button>
+            {!queryString.get('tab') && <h1>My profile</h1>}
+            {queryString.get('tab') === 'Stats' && <h1>Stats tab</h1>}
+            {queryString.get('tab') === 'Counterparties' && <h1>Counterparties tab</h1>}
+            {queryString.get('tab') === 'Payment methods' && <h1>Payment methods tab</h1>}
+            {queryString.get('form') === 'Payment method form' && <h1>Payment method form</h1>}
+        </div>
+    );
+};
+
+const MockHome = () => <h1>Home</h1>;
+
+const MockAppContent = () => {
+    return (
+        <Switch>
+            <Route component={MockHome} exact path='/' />
+            <Route component={MockBuySell} path='/mock-buy-sell' />
+            <Route component={MockMyProfile} path='/mock-my-profile' />
+        </Switch>
+    );
+};
+
+const mockReplace = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: () => ({
+        push: jest.fn(),
+        replace: mockReplace,
+    }),
+}));
+
+let windowLocationSpy: jest.SpyInstance<Location, []>;
+
+describe('useQueryString', () => {
+    beforeEach(() => {
+        windowLocationSpy = jest.spyOn(window, 'location', 'get');
+    });
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+    test('setting new query strings', () => {
```

Comment:
nit: lets follow the same convention across our test files 🙏 

```suggestion
    it('should set new query strings', () => {
```

Reply:
lets do the same for all the other test cases below 🙏